doc/sphinx/source/recipes/recipe_cmorizers.rst

```diff
@@ -0,0 +1,94 @@
+.. _recipe_cmorizers:
+
+Cmorizers recipes
```

Comment:
```suggestion
CMORizer recipes
```
Please write "CMOR" as all uppercase characters everywhere, because it is a an acronym